bloxid/v0.go

```diff
@@ -181,6 +277,9 @@ type V0Options struct {
 	EntityDomain string
 	EntityType   string
 	extrinsicID  string
+	hashid       int
+	hashidSalt   string
+	idType       string
```

Comment:
```suggestion
	scheme       string
```

please also add Scheme() to interface.go